discord/client.py

```diff
@@ -1133,34 +1136,68 @@ def _check(*args):
 
     # event registration
 
-    def event(self, coro: Coro) -> Coro:
+    def event(self, event: str) -> Coro:
```

Comment:
sorry if I'm misunderstanding, but wouldn't this be a breaking change? Maybe keep how it currently is and add support on `@listen` as well

Reply:
This is used to be a decorator, meaning that it was used like this:
```py
import discord

client = discord.Client()

@client.event
async def on_ready():
    pass
```

Meaning that users could not use the argument as a keyword argument

Reply:
Doesn't ``@listen`` already have similar functionality? It allows you to pass an event name as parameter or no parameters , but still with brackets, for it to use the handler name.

Reply:
Is it ok like this?